webdriver-extension/index.bs

```diff
@@ -0,0 +1,553 @@
+    <pre class="idl">
+      dictionary UncalibratedMagnetometerReadingValues {
+        required double? x;
+        required double? y;
+        required double? z;
+        required double? xBias;
+        required double? yBias;
+        required double? zBias;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Absolute Orientation Sensor ### {#mock-absolute-orientation-sensor}
+The <dfn for="MockSensorType" enum-value>"absolute-orientation"</dfn> type is the mock sensor type
+associated with the usage of the <a>AbsoluteOrientationSensor</a> interface.
+<dl>
+  <dt>[=Mock Sensor Reading Values=]</dt>
+  <dd>
+    <pre class="idl">
+      dictionary AbsoluteOrientationReadingValues {
+        required FrozenArray&lt;double>? quaternion;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Relative Orientation Sensor ### {#mock-relative-orientation-sensor}
+The <dfn for="MockSensorType" enum-value>"relative-orientation"</dfn> type is the mock sensor type
+associated with the usage of the <a>RelativeOrientationSensor</a> interface.
+<dl>
+  <dt>[=Mock Sensor Reading Values=]</dt>
+  <dd>
+    <pre class="idl">
+      dictionary RelativeOrientationReadingValues : AbsoluteOrientationReadingValues {
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Geolocation Sensor ### {#mock-geolocation-sensor}
+The <dfn for="MockSensorType" enum-value>"geolocation"</dfn> type is the mock sensor type
+associated with the usage of the <a>GeolocationSensor</a> interface.
+<dl>
+  <dt>[=Mock Sensor Reading Values=]</dt>
+  <dd>
+    <pre class="idl">
+      dictionary GeolocationReadingValues {
+        required double? latitude;
+        required double? longitude;
+        required double? altitude;
+        required double? accuracy;
+        required double? altitudeAccuracy;
+        required double? heading;
+        required double? speed;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Proximity Sensor ### {#mock-proximity-sensor}
+The <dfn for="MockSensorType" enum-value>"proximity"</dfn> type is the mock sensor type
+associated with the usage of the <a>ProximitySensor</a> interface.
+<dl>
+  <dt>[=Mock Sensor Reading Values=]</dt>
+  <dd>
+    <pre class="idl">
+      dictionary ProximityReadingValues {
+        required double? distance;
+        required double? max;
+        required boolean? near;
+      };
+    </pre>
+  </dd>
+</dl>
+
+Extensions {#webdriver-extensions}
+=====================
+
+## Commands ## {#mock-sensor-commands}
+
+### Create mock sensor ### {#create-mock-sensor-command}
+
+<table>
+  <tbody>
+    <tr>
+      <th>HTTP Method</th>
+      <th><a lt="extension command URI template">URI Template</a></th>
+    </tr>
+    <tr>
+      <td>POST</td>
+      <td>/session/{session id}/sensor</td>
+    </tr>
+  </tbody>
+</table>
+
+The <dfn>create mock sensor</dfn> <a>extension command</a> simulates user
+creation of a [=mock sensor=].
+
+The <a>remote end steps</a> are:
+1. Let |configuration| be the |configuration| argument, <a>converted to an IDL value</a>
```

Comment:
The data from a WebDriver POST command body should be referred to as "parameters".